x-pack/plugins/ml/public/application/services/ml_api_service/trained_models.ts

```diff
@@ -288,6 +289,14 @@ export function trainedModelsApiProvider(httpService: HttpService) {
         version: '1',
       });
     },
+
+    getModelsDownloadStatus() {
+      return httpService.http<Record<string, ModelDownloadState> | undefined>({
```

Comment:
```suggestion
      return httpService.http<Record<string, ModelDownloadState>>({
```

Reply:
Updated in https://github.com/elastic/kibana/pull/184906/commits/d0db2fb9c8d78514a2f7cf02a8487c07150b2568